### ISSUE TITLE
Endianness Converter

### ISSUE THREAD

**Issue by dwilkie** [twitter/cloudhopper-commons#8]
Would it be ok if I submitted a PR which adds the following method to `CharsetUtil`? I the SMSC that I'm connecting too uses LittleEndian for UCS-2

```java
  static public byte[] getEndianBytes(byte [] sourceBytes, ByteOrder destByteOrder) {
    ByteBuffer sourceByteBuffer = ByteBuffer.wrap(sourceBytes);
    ByteBuffer destByteBuffer = ByteBuffer.allocate(sourceBytes.length);

    destByteBuffer.order(destByteOrder);
    while(sourceByteBuffer.hasRemaining()) {
      destByteBuffer.putShort(sourceByteBuffer.getShort());
    }

    return destByteBuffer.array();
  }
```

**Comment by jjlauer** [comment]
Would be happy to consider the PR.  I'd just throw out a couple ideas:

1. A unit test for it would be good -- as well as the implementing code.
2. Should it be considered a new charset with a specific name?

-Joe

**Comment by dwilkie** [comment]
@jjlauer see: https://github.com/twitter/cloudhopper-commons/pull/9

**Comment by dwilkie** [comment]
Closed by #9
